modules/doc/content/newsletter/2021_03.md

```diff
@@ -1 +1,47 @@
 # MOOSE Newsletter (March 2021)
+
+
+## MooseDocs and CIVET Paper Published
+
+We published a paper regarding are SQA practices and the associate the use of MooseDocs and CIVET.
+If you are using CIVET for testing and/or MooseDocs for documentation, please reference the
+paper listed here: [citing.md#testing-and-documentation].
+
+## Early Construction of Subapps of MultiApps
+
+A design change on MultiApp has been made recently. With this change, MOOSE constructs
+the sub-applications (subapps) of MultiApps by default right after when MOOSE adds the
+MultiApps to the main application. Previously, the construction of subapps happens
+within the initial setup of the main application after the execution of all Actions.
+Derived MultiApps can set the private parameter "use_positions" of MultiApp to *false*
+to override this behavior and delay the construction as before, which is a feature
+required by the sampler MultiApps in the MOOSE stochastic module. This change should
+not affect the calculations with MultiApps but allows the main application to inquiry
+the subapps for setting up its objects such as kernels, boundary conditions, etc.
+accordingly. The design is documented in [MultiApps](syntax/MultiApps/index.md) along
+with some other document updates related with MultiApp.
+=======
+>>>>>>> 26afefa6e4 (Mar. 2021 news)
```

Comment:
Looks like a leftover from rebasing...

Reply:
@fdkong Can you fix this when you add your items.

Reply:
I will